Jenkinsfile

```diff
@@ -6,10 +6,24 @@ pipeline {
     environment {
         IMAGE_STREAM_TAG = "latest"
         REGISTRY = "docker-registry.engineering.redhat.com"
-        PROJECT = "lifecycle"
+        DEV_PROJECT = "lifecycle"
+        STAGE_PROJECT = "lifecycle"
         STAGE_URI = "insecure://openshift-ait.e2e.bos.redhat.com:8443"
+        DEV_URI = "insecure://osemaster.sbu.lab.eng.bos.redhat.com:8443"
```

Comment:
Should we put all of these into env.VAR that is passed in from the OpenShift BC?

Reply:
Or maybe these will be overridden by those in the BC

Reply:
AFICT, openshift BC env vars are mapped as both params.KEY and regular environment vars.

Reply:
@aweiteka let me test this